src/components/Questionnaire/QuestionTypes/EncounterQuestion.tsx

```
@@ -239,33 +239,50 @@ export function EncounterQuestion({
-
+  const isDischarged = encounter.status === EncounterStatus.DISCHARGED;
   return (
     <div className="space-y-6">
       <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
         {/* Basic Details */}
         <div className="space-y-2">
           <Label>{t("encounter_status")}</Label>
+          <div
+            data-testid="encounter-status"
+            aria-label={t("current_encounter_status")}
+          >
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)